projects/angular2-query-builder/src/lib/query-builder/query-builder.component.html

```diff
@@ -153,8 +153,11 @@
                     and
                     <input [ngClass]="getClassNames('inputControl')" [(ngModel)]="rule.value[1]" (ngModelChange)="changeInput()" [disabled]="disabled" type="number">
                 </ng-container>
-                <input [ngClass]="getClassNames('inputControl')" [(ngModel)]="rule.value" (ngModelChange)="changeInput()"
-                  [disabled]="disabled" *ngSwitchCase="'boolean'" type="checkbox">
+                <select [ngClass]="getClassNames('inputControl')" [(ngModel)]="rule.value" (ngModelChange)="changeInput()"
+                  [disabled]="disabled" *ngSwitchCase="'boolean'">
+                  <option [ngValue]="true">true</option>
```

Comment:
Nitpick, could you see with Danny if he prefers lowercase or uppercase on the options? true vs True vs TRUE

Reply:
Check with him and changed the case as he suggested.